deploy/docker-compose-with-service.yml

```diff
@@ -0,0 +1,150 @@
+version: '3.9'
+services:
```

Comment:
I think we should just have a single compose file that deploys only the sync service. Everything else should be checkout of from the main negotiator repository (like the compose that deploys the application). If you need to change some settings of the oidc mock just open a PR in the NEgoitator repo

Reply:
Fixed in [a01cf2b](https://github.com/BBMRI-ERIC/directory-negotiator-sync/pull/1/commits/a01cf2bfa2fe7a7a8a0924685aa15dd90f812787)